libs/moment-adapter/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,175 @@
+import { Rule, SchematicContext, Tree, chain, SchematicsException } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { getPackageVersionFromPackageJson, hasPackage } from '../utils/package-utils';
+import { addPackageJsonDependency, NodeDependency, NodeDependencyType } from '@schematics/angular/utility/dependencies';
+import { hasModuleImport } from '../utils/ng-module-utils';
+import { WorkspaceSchema } from '@schematics/angular/utility/workspace-models';
+
+import { defaultFontStyle } from './styles';
+import {
+    addModuleImportToModule,
+    findModuleFromOptions
+} from '@angular/cdk/schematics';
+
+const browserAnimationsModuleName = 'BrowserAnimationsModule';
+const noopAnimationsModuleName = 'NoopAnimationsModule';
+const fdStylesIconPath = 'node_modules/fundamental-styles/dist/icon.css';
+
+export function ngAdd(options: any): Rule {
+    return chain([
+        addDependencies(),
+        endInstallTask(),
+        addAnimations(options),
+        addStylePathToConfig(options),
+        addFontsToStyles(options)
```

Comment:
do we need animations/styles/fonts for this package?

Reply:
removed